### PR TITLE
Update monitoring.html.md

### PR DESCRIPTION
**Pull request by hashitop**
The collected metrics are not included

## Labels

- [X] inaccuracy
- [X] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

**Comment by hashitop** [comment]
The reference would be this, https://help.replicated.com/docs/native/packaging-an-application/support-bundle/#default-support-files

File: content/source/docs/enterprise/admin/monitoring.html.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ In addition to health-check monitoring, we recommend monitoring standard server
 
 ## Internal Monitoring
 
-Beginning in version 201811-1, Terraform Enterprise includes internal monitoring of critical application metrics using a statsd collector. The resulting metrics are included with the [diagnostic bundles provided to HashiCorp Support](../support/index.html). If the Terraform Enterprise instance is already running a collector on the same port, Terraform Enterprise may not start up correctly due to the conflict, and logs will indicate:
+Beginning in version 201811-1, Terraform Enterprise includes internal monitoring of critical application metrics using a statsd collector. If the Terraform Enterprise instance is already running a collector on the same port, Terraform Enterprise may not start up correctly due to the conflict, and logs will indicate:
```
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/monitoring.html.md:36]: I'm just editing for flow. Let me know if you don't like this or it materially changes the meaning! 

```suggestion
Beginning in version 201811-1, Terraform Enterprise uses a statsd collector to monitor critical application metrics. Terraform Enterprise produces an error during start up if the instance is already running a collector on the same port:
```

**Review by laurapacilio — APPROVED** [review]
Left a suggested edit for flow - thank you!

File: content/source/docs/enterprise/admin/monitoring.html.md (hunk 1 above)
Comment (hashitop) [reply content/source/docs/enterprise/admin/monitoring.html.md:36]: Thank you! I think it is best to have @sudomateo and @justinretzolk across this.
I am not certain about the intent of this particular section. The PR was only aimed to clarify that the stats are not included in the support bundle.

Comment (hashitop) [reply content/source/docs/enterprise/admin/monitoring.html.md:36]: Probably inaccurate to use the term clarify, it is to remove the ambiguity and misunderstanding.

Comment (sudomateo) [reply content/source/docs/enterprise/admin/monitoring.html.md:36]: I like the suggestion!

Comment (laurapacilio) [reply content/source/docs/enterprise/admin/monitoring.html.md:36]: @hashitop - now that Matthew has weighed in on this, do you feel comfortable making this change? :-)

**Pull request merged**